### ISSUE TITLE
bug: nested tree views has odd arrow key behavior

### ISSUE THREAD

**Issue by scomea** [microsoft/fast#3886]
**Describe the bug; what happened?**
Using arrow keys to navigate nested tree view has odd behaviors.
- sometimes pressing an arrow key does nothing, then pressing it again moves focus
- if you navigate to an item by pressing the right arrow key you should be able to go back by pressing left arrow key

**What are the steps to reproduce the issue?**

Go to https://explore.fast.design/components/fast-tree-view
click on tree item 1 to focus on it
press right arrow key, item opens
press right arrow key, focus to nested child
press rigth arrow key, nothing happens (?)
press right arrow key, focus to tree item 2
press left arrow key, nothing happens

**What behavior did you expect?**
Consistent and predictable arrow key navigation

**Comment by chrisdholt** [comment]
Thanks @scomea, let's be sure that our arrow behavior follows this: https://w3c.github.io/aria-practices/#TreeView
